docs/stdlib/doc/fs.md

```diff
@@ -0,0 +1,81 @@
+## `change_owner`
+```ab
+pub fun change_owner(user: Text, path: Text): Bool 
+```
+
+Changes the owner of a file.
+
+If the file doesn't exist, it returns `false`
+
+
+## `create_dir`
+```ab
+pub fun create_dir(path: Text): Null 
+```
+
+Creates a directory with all parent directories as required.
+
+
+## `create_symbolic_link`
+```ab
+pub fun create_symbolic_link(origin: Text, destination: Text): Bool 
+```
+
+Creates a symbolic link.
+
+If the file doesn't exist, it returns a boolean and prints a message.
+
+
+## `dir_exist`
+```ab
+pub fun dir_exist(path) 
+```
+
+Checks if a directory exists.
+
+
+## `file_append`
+```ab
+pub fun file_append(path, content) 
+```
+
+Appends content to a file.
+
+Doesn't check if the file exists.
```

Comment:
what would happen if the file doesn't exist? or a permission error

not clear using what method does it use to append content

not clear what does it return